src/format_sexp.mli

```diff
@@ -0,0 +1,7 @@
+open Import
```

Comment:
What do you think of calling this file `Format_dune_lang`?

Reply:
Sounds good to me.